sdk/Sdk.Generators/Properties/launchSettings.json

```diff
@@ -0,0 +1,8 @@
+{
+  "profiles": {
+    "SourceGenDebug": {
+      "commandName": "DebugRoslynComponent",
```

Comment:
Used to debug against a target project at compilation time - I think this is worth keeping.

Reply:
Debating on how to add the other component for this ([example setup](https://github.com/JoanComasFdz/dotnet-how-to-debug-source-generator-vs2022)) in the `FunctionApp.csproj` file since it can make other debugging things work poorly with another project reference. Other options:

1. Add the necessary component but commented out w/ additional warnings/instructions.
2. Add SourceGen debugging instructions to README.md or Wiki, leaving just this launch profile here and the other component in the documentation for devs to copy/paste over.